src/main/scala/firrtl/Utils.scala

```diff
@@ -232,7 +234,18 @@ object Utils extends LazyLogging {
   def indent(str: String) = str replaceAllLiterally ("\n", "\n  ")
 
   implicit def toWrappedExpression (x:Expression): WrappedExpression = new WrappedExpression(x)
-  def ceilLog2(x: BigInt): Int = (x-1).bitLength
+  def getSIntWidth(s: BigInt): Int = s.bitLength + 1
+  def getUIntWidth(u: BigInt): Int = u.bitLength
+  private val zdec1 = """([+\-]?[0-9]\d*)(\.[0-9]*[1-9])(0*)""".r
```

Comment:
Quibble: Shouldn't the regexes start with capitals

Reply:
Is that like a stylistic thing?